testgrid/config/config.yaml

```diff
@@ -188,12 +188,8 @@ test_groups:
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-federation
 - name: kubernetes-e2e-gce-federation-serial
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-federation-serial
-- name: kubernetes-e2e-gci-gce-federation
-  gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce-federation
-- name: kubernetes-e2e-gce-federation-release-1.4
-  gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-federation-release-1.4
-- name: kubernetes-e2e-gce-federation-release-1.5
-  gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-federation-release-1.5
+- name: kubernetes-e2e-gce-federation-release-1.6
+  gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-federation-release-1.6
```

Comment:
ahh, there's already a testgroup named `ci-kubernetes-e2e-gce-federation-release-1.6`, keep this or the other.

the name is not very consistent here :-|